cogboard-app/src/main/kotlin/com/cognifide/cogboard/storage/docker/Validation.kt

```diff
@@ -0,0 +1,59 @@
+package com.cognifide.cogboard.storage.docker
+
+import com.cognifide.cogboard.CogboardConstants
+import io.vertx.core.json.JsonObject
+import java.lang.ClassCastException
+
+class Validation {
+    companion object {
+        fun validate(config: JsonObject): Boolean {
+            val boards = config.getJsonObject("boards")?.getJsonObject("boardsById")
+            return boards != null && validateBoards(boards)
+                    && config.getJsonObject("widgets")?.getJsonObject("widgetsById") != null
+        }
+
+        private fun validateBoards(boards: JsonObject): Boolean {
+            val titles = mutableSetOf<String>()
+
+            return try {
+                boards.fieldNames().stream().allMatch { id ->
+                    val board = boards.getJsonObject(id)
+                    checkColumnsRange(board)
+                            && checkTitleLength(board)
+                            && checkTitleUnique(board, titles)
+                            && checkAutoSwitchInterval(board)
+                }
+            } catch (e: ClassCastException) {
+                false
```

Comment:
please at least comment when this exception can happen, now it looks ugly - not sure what can be a problem

I suggest to move this catch inside [particular validation methods so it becomes obvious what it is actually catchin.

Reply:
Another idea, maybe you could introduce a class that represents the board instead of using raw JSON object? Then JSON mapper would have perform some validation for you - types matching, not nulls, etc..